soroban-sdk/src/storage.rs

```diff
@@ -76,18 +76,59 @@ impl Storage {
         Storage { env: env.clone() }
     }
 
+    /// Storage for data that can stay in the ledger forever until deleted.
+    ///
+    /// Persistent entries might be 'evicted' from the ledger if they run out
+    /// of the rent balance. However, evicted entries can be restored and also
+    /// they cannot be recreated. This means that semantically these entries
+    /// behave 'as if' they were stored in the ledger forever.
```

Comment:
Can we use the term expired only? Eviction is a bit of an implementation detail. A contract really only deals with live and expired, and contracts don't see the difference between expired and evicted. Suggest:
```suggestion
    /// Persistent entries might expire and be removed from the ledger if they run out
    /// of the rent balance. However, expired entries can be restored and
    /// they cannot be recreated. This means these entries
    /// behave 'as if' they were stored in the ledger forever.
```

Reply:
Ok, I just thought it's the other way around and 'evicted' si _the_ term.